### PR TITLE
Prepare for release.

### PR DESCRIPTION
**Pull request by Evelyn-M**
This is for the next release (v1.1.1).
Changelog, readme, pyproject.toml and index are amended.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,24 @@
 # Changelog
+## v1.1.1
+
+Release date: 2023-11-24
+
+### Description
+
+Minor bug fixes and improvements to readthedocs. 
+
```
Comment (peanutfun) [inline CHANGELOG.md:9]:
```suggestion

### Added

* Option to enable module-wide logging with `osm_flex.enable_logs()`.
```

**Review by peanutfun — CHANGES_REQUESTED** [review]
Looks great. Do we want to mention the new logging option in the CHANGELOG, too?

File: CHANGELOG.md (hunk 1 above)
Comment (Evelyn-M) [reply CHANGELOG.md:9]: Good point!

**Review by peanutfun — CHANGES_REQUESTED** [review]
👏

**Review by peanutfun — APPROVED** [review]
👏

**Pull request merged**